python/tvm/micro/model_library_format.py

```diff
@@ -55,17 +55,19 @@ def _populate_codegen_dir(mod, codegen_dir: str):
 
     mod_indices = {"lib": 0, "src": 0}
     host_codegen_dir = os.path.join(codegen_dir, "host")
+    lib_name = f"{model_name}_lib" if model_name else "lib"
+
     for dso_mod in dso_modules:
         if dso_mod.type_key == "c":
             index = mod_indices["src"]
             mod_indices["src"] += 1
             parent_dir = os.path.join(host_codegen_dir, "src")
-            file_name = os.path.join(parent_dir, f"lib{index}.c")
+            file_name = os.path.join(parent_dir, f"{lib_name}{index}.c")
```

Comment:
Mmmm... Not sure I follow :) To be clear, I am adding a prefix to the lib names so that multiple source files can be in the same folder. Otherwise the user would need to compile each library and manually copy/paste the `lib*.c` files in a common folder with different names.

Reply:
yeah i like this idea, but we should bump the [version](https://github.com/apache/tvm/blob/main/python/tvm/micro/model_library_format.py#L225) in the metadata to indicate a format change. since MLF is new and not used much yet, i don't think this is consequential, and would rather get in the practice of doing this now rather than trying to judge some more confusing criteria later on.